crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -884,13 +893,13 @@ impl Backend {
                                         // assuming max 5 args
                                         Some(token) => {
                                             node_info!(
-                                                "    Error: reverted with custom error: {:?}",
+                                                "    Error: reverted with custom error (1): {:?}",
```

Comment:
```suggestion
                                                "    Error: reverted with custom error: {:?}",
```

Reply:
FYI the point of this is to differentiate the source of the error. anvil's errors are sometimes formatted exactly the same but originate in different places.